R/SimInf_model.R

```diff
@@ -1392,10 +1392,20 @@ setMethod("run",
                   name <- as.character(class(model))
 
                   ## The model C run function
-                  run_fn <- paste0(name, "_run")
-
-                  ## Create expression to parse
-                  expr <- ".Call(run_fn, model, threads, solver, PACKAGE = 'SimInf')"
+                  if(startsWith(name, "SimInf-")) {
```

Comment:
Since SimInf has `Depends: R(>= 3.1)` in the DESCRIPTION file, `startsWith` must be replaced with, for example, `substr("SimInf-model-", 1, 7) == "SimInf-"`. I think `startsWith` was introduced in ~ R 3.3